t/Currency.t

```diff
@@ -13,12 +13,6 @@ ddg_spice_test(
     [
         'DDG::Spice::Currency'
     ],
-    'canada dollar' => test_spice(
```

Comment:
I think this is a case where we should still show the IA. It's pretty clear what they're inquiring about, but when it's a 3-letter abbreviation, it's harder to know because it has multiple meanings